site/content/docs/private-locations/checkly-agent-guide.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Installing Checkly Agent
+weight: 52
+menu:
+  docs:
+    parent: "Private Locations"
+aliases:
+- "/docs/private-locations/checkly-agent-guide/"
+---
+
+Note: The Checkly Agent currently only supports x86/AMD64 architecture and not ARM64. This means Apple Mac M1 systems are not currently supported.
+
+## About Checkly Agent
+
+**You must have the Owner or Admin role on your Checkly account to create private locations.**
+
+The Checkly Agent is a container-based tool that enables private locations in Checkly. The agent runs on your infrastructure in a container and runs checks on behalf of the Checkly application. Since the agent is local to your infrastructure, checks can run against any application accessible from the agent via HTTPS or HTTP.
+
+Here's how private locations work:
+
+A private location is similar to our existing public locations, but you have control over it. A private location can be in your own segregated network,  on-premises or cloud-based. You need one or more Checkly agents installed in that location/network with access to your applications. Checks are configured and scheduled in the Checkly cloud-based web UI as usual, but selecting a private location runs the check on the agent(s) in your infrastructure. Agents are stateless and ephemeral. As long as you have at least one operational agent, checks will run in your private location. Adding more agents will distribute the load and improve resilience and performance automatically.
+
+Here are the requirements before you get started:
```

Comment:
This part also exists in the "Getting started with private locations" page. This page feels more like "Optimizing Checkly Agent" rather than "Installing Checkly Agent" btw

Reply:
No, the plan is to keep both.

Reply:
I think "Installing Checkly Agent" and "Getting Started with Private Locations" are pretty similar topics, and the two pages seem to have quite a bit of overlap.

I think that it would make sense to restructure things and have a more clear distinction between all of the pages.